bindings/bindingtester/tests/api.py

```diff
@@ -600,6 +600,14 @@ def generate(self, args, thread_number):
                 instructions.append(op)
             elif op == 'TENANT_CLEAR_ACTIVE':
                 instructions.append(op)
+            elif op == 'TENANT_LIST_NAMES':
+                instructions.push_args(b'', b'\xff', 10000)
```

Comment:
You should generate these parameters randomly so that we try different combinations of values. The range read operation would probably have some similar logic that you could reference.